targets/targets.json

```diff
@@ -6802,6 +6802,71 @@
         "release_versions": ["5"],
         "detect_code": ["3703"]
     },
+    "FAMILY_Apollo3": {
+        "inherits": ["Target"],
+        "core": "Cortex-M4F",
+        "features": ["BLE"],
+        "default_toolchain": "GCC_ARM",
+        "supported_toolchains": [
+            "GCC_ARM", 
+            "ARMC6"
+        ],
+        "public": false,
+        "extra_labels": [
+            "Ambiq_Micro",
+            "Apollo3",
+            "CORDIO"
+        ],
+        "device_has": [
+            "MPU",
+            "USTICKER",
+            "SERIAL",
+            "INTERRUPTIN",
+            "LPTICKER",
+            "STDIO_MESSAGES",
+            "FLASH",
+            "SPI",
+            "I2C"
+        ],
+        "components": [
+            "FLASHIAP"
+        ],
+        "macros": ["CORDIO_ZERO_COPY_HCI",
```

Comment:
this should be on the new line as the rest of the them follow so `[` and `"CORDIO..." on the new line

Reply:
done